### ISSUE TITLE
How to disable memcached?

### ISSUE THREAD

**Issue by cocowalla** [haiwen/seafile-docker#280]
I've just upgraded a 6.x Seafile install to the latest version, on a server with only 2GB of memory - and so can't afford to run memcached.

I've removed it from my Docker Compose file, but when I run the gc process, I see thousands upon thousands of these lines:

```
2021-12-10 10:08:17 ../../common/obj-cache.c(111): Failed to set e9e19426-197d-4b8a-9f40-7a506f84b6e9-f3a630b9a15f91e0d31582d14e5b1959340b8643 to memcached: SERVER HAS FAILED AND IS DISABLED UNTIL TIMED RETRY
```

So at the least, the GC script is expecting memcached to be running - I don't know if maybe other parts of Seafile are too.

Is there a configuration switch I'm missing where I can disable Memcached altogether, so no part of Seafile will try to use it?

**Comment by dimejo** [comment]
Haven't tried this myself and just guessing but there's an entry for memcached in `conf/seahub_settings.py`. Maybe removing it helps?

**Comment by cocowalla** [comment]
Nope, I don't have a memcached or `CACHES` entry in my seahub settings file at all.
